tests/integration/sugar/test_transaction.py

```diff
@@ -455,3 +457,134 @@ async def test_reliable_submission_no_last_ledger_sequence(self, client):
         signed_payment_transaction = await sign(payment_transaction, WALLET)
         with self.assertRaises(XRPLReliableSubmissionException):
             await send_reliable_submission(signed_payment_transaction, client)
+
+
+class TestSubmitAndWait(IntegrationTestCase):
+    @test_async_and_sync(
+        globals(),
+        [
+            "xrpl.transaction.submit_and_wait",
+            "xrpl.account.get_next_valid_seq_number",
+            "xrpl.ledger.get_fee",
+        ],
+    )
+    async def test_submit_and_wait_simple(self, client):
+        WALLET.sequence = await get_next_valid_seq_number(ACCOUNT, client)
```

Comment:
Why do we need to get the sequence in these tests? (Shouldn't that already be part of `autofill`?)

Reply:
(If this is to help test cases run in parallel, I'd recommend writing at least one test case where you create a fresh wallet and autofill the sequence number because that's an important part of the workflow)